### PR TITLE
card design made readable

### PR DESCRIPTION
**Pull request by sre-harine**
(( Relates to #96 ))

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -5,13 +5,15 @@ import clsx from 'clsx';
...
+    <div className="card filter brightness-105 shadow-xl">
       {backgroundImageSource && (
         <figure>
           <Image
             alt={backgroundImageAltText}
-            layout="fill"
+            layout="fixed"
             src={backgroundImageSource}
+            width="350"
+            height="250"
```
Comment (RedFox0x20) [inline components/Card.js:16]: As per linting please sort these attributes alphabetically;
alt, height, layout, src, width.

**Review by RedFox0x20 — CHANGES_REQUESTED** [review]
Thanks for the contribution, we will be able to review the visual difference once the required changes are made. Thanks for keeping it simple and easy to review!

(( Relates to #96 ))

**Review by RedFox0x20 — APPROVED** [review]
The images are much cleaner like this and provides the site with a more modern aesthetic while maintaining a mobile-friendly style. Text contrast is greatly improved!

Thanks for the contribution!

File: components/Card.js [hunk 2]
```diff
@@ -5,13 +5,15 @@ import clsx from 'clsx';
 
 function Card({ backgroundImageSource, backgroundImageAltText, children }) {
   return (
-    <div className="card filter brightness-105 shadow-xl image-full">
+    <div className="card filter brightness-105 shadow-xl">
```
Comment (ozer619) [inline components/Card.js:8]:
```suggestion
    <div className="card filter brightness-105 shadow-xl  style={{
        boxShadow:
          'rgba(14, 30, 37, 0.1) 0px 0px 4px 0px, rgba(14, 30, 37, 0.15) 0px 0px 16px 0px;',
      }}">
```
Modifies box-shadow.

File: components/Card.js [hunk 3]
```diff
@@ -5,13 +5,15 @@ import clsx from 'clsx';
...
       {backgroundImageSource && (
         <figure>
           <Image
             alt={backgroundImageAltText}
-            layout="fill"
+            height="250"
```
Comment (ozer619) [inline components/Card.js:13]:
```suggestion
            height="280"
```

File: components/Card.js [hunk 4]
```diff
@@ -5,13 +5,15 @@ import clsx from 'clsx';
...
+            layout="fixed"
             src={backgroundImageSource}
+            width="350"
```
Comment (ozer619) [inline components/Card.js:16]:
```suggestion
            width="400"
```

**Review by ozer619 — CHANGES_REQUESTED** [review]
Hey, @sre-harine your Pr looks great!! just some small changes regarding the card UI.
Thanks For your contribution :)

File: components/Card.js (hunk 2 above)
Comment (ozer619) [reply components/Card.js:8]: I played with it a little and created this custom combination.

Comment (marekrozmus) [reply components/Card.js:8]: If you'll add that then please do not use inline styles.

**Review by marekrozmus — CHANGES_REQUESTED** [review]
I got just two visual issues:
- take a look at top of cards if background color is not white - the image comes out of the card
problem
- the yellow color of link needs to be also updated - not it is barely visible:
problem2

File: components/Card.js [hunk 5]
```diff
@@ -5,13 +5,15 @@ import clsx from 'clsx';
...
   return (
-    <div className="card filter brightness-105 shadow-xl image-full">
+    <div className="card filter brightness-105 shadow-xl">
       {backgroundImageSource && (
         <figure>
           <Image
             alt={backgroundImageAltText}
-            layout="fill"
+            height="250"
+            layout="fixed"
```
Comment (LowLifeArcade) [inline components/Card.js:14]: Hey @ozer619. So I played around more after we talked and `layout="responsive"` seems to do the trick of keeping the aspect ratio here. Seems like an appropriate term! 😁

File: components/Card.js (hunk 2 above)
Comment (LowLifeArcade) [reply components/Card.js:8]: Yeah, this is a great improvement to the look! We talked about moving it into a styled jsx element but I'm not sure if it has that big of a performance benefit as I had thought as per the [documentation](https://nextjs.org/docs/basic-features/built-in-css-support#css-in-js). It's an option, but this seems to work just as well. Regardless, it's a nice look.

**Review by LowLifeArcade — CHANGES_REQUESTED** [review]
Things are looking good. I think just a small change is all that's needed to line 14.

**Review by LowLifeArcade — APPROVED** [review]
Everything is ready to go. Thank you @sre-harine 🥳

**Pull request merged**